nonlingeo_precice.c

```diff
@@ -1114,10 +1114,10 @@ void nonlingeo_precice(double **cop, ITG *nk, ITG **konp, ITG **ipkonp, char **l
 	  
 	  memcpy(&vini[0],&vold[0],sizeof(double)*mt**nk);
 
-	  if( Precice_IsWriteCheckpointRequired() )
+	  if( Precice_IsWriteCheckpointRequired() || simulationData.coupling_explicit)
```

Comment:
This would then almost always be true, right? I understnad that you always want checkpoints to be stored (to trigger some internal update).

Reply:
For explicit coupling, yes.